src/search.cpp

```diff
@@ -1250,6 +1250,11 @@ namespace {
       {
           Depth r = reduction<PvNode>(improving, depth, moveCount);
 
+#ifdef ANTI
+          if (pos.is_anti() && pos.can_capture())
+              r -= r ? ONE_PLY : DEPTH_ZERO;
+          else
```

Comment:
To the first point, I don't know how else to simply test for stability, and while simple it doesn't sound easy.

Last time I checked the `crank` top computer player list, the strongest engines used PN^2 in combination with other algorithms.  What that means is that changes which bring Stockfish closer toward emulating (though not explicitly implementing) a proof-number search & evaluation should gain Elo (at least until upstream [significantly changes search/qsearch](a18c2c2c3f38070cb897a8a28cf1d6d45236ded4), and/or [adds or removes evaluation factors](b24bd762b2aff6cac30e1ae673775ea6f6aa8d11) and we need to understand where the bug is and fix it).